pages/the-lab.vue

```diff
@@ -40,7 +41,13 @@ useHead({
       </h1>
       <div class="flex mt-5" style="gap: 1rem">
         <v-flexible-link to="/the-lab" raw>
-          <Button class="p-button-rounded" label="Become a member" />
+          <Button
+            class="p-button-rounded"
+            label="Become a member"
+            @click="
+              gaEvent('Click Tracking', 'The Lab Hero CTA', 'Become a member')
+            "
+          />
         </v-flexible-link>
         <v-flexible-link to="https://members.radiolab.org/request_access" raw>
```

Comment:
do we need GA here?

Reply:
I don't think so - it will automatically track these links. With the "become a member" buttons, the RL team specifically asked that we track where exactly the buttons are on the page. I don't think they care about the sign in buttons/links, they care about _where_ to put the become a member CTAs

Reply:
cool